### ISSUE TITLE
I added this library but using same steps but not working in IOS

### ISSUE THREAD

**Issue by shashitechvalens** [smarkets/react-native-paypal#83]
MergeSwiftModule normal x86_64
        CompileC /Users/shashi/Library/Developer/Xcode/DerivedData/MedGlow-cxviahozxnzcxmcmkfoflokikqcc/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/RNPaypal.build/Objects-normal/x86_64/RNPaypal.o /Users/shashi/Desktop/trial/medglow_backup_last26feb/MedGlow/node_modules/react-native-paypal/ios/RNPaypal.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler

i have xcode 11.3

**Comment by nzankich** [comment]
@shashitechvalens could you paste your error?

It may help to compare your app against the example one to make sure everything is set up correctly

**Comment by shashitechvalens** [comment]
1 :- i installed library
2. did entry in pood file
  pod 'RNPaypal', :path => '../node_modules/react-native-paypal'
  pod 'Braintree', '~> 4.19.0'
3.pod install
4.then run
But when i run i'm getting response
The following build commands failed:
        CompileC /Users/shashi/Library/Developer/Xcode/DerivedData/MedGlow-cxviahozxnzcxmcmkfoflokikqcc/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/Flipper-Folly.build/Objects-normal/x86_64/SSLSessionImpl.o /Users/shashi/Desktop/trial/medglow_backup_last26feb/MedGlow/ios/Pods/Flipper-Folly/folly/ssl/detail/SSLSessionImpl.cpp normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
Its Architecture error.
Please help i'm stuck here since 2 days.

**Comment by nzankich** [comment]
do you have the actual error though?  what does the failed build command say?

also did you get a chance to try out the example app and see if that builds?

**Comment by shashitechvalens** [comment]
Yes, i tried with new fresh example aslo but still the same error i'm getting on XCODE 11. unbale to build or run
normal/x86_64/SSLSessionImpl.o /Users/shashi/Desktop/trial/medglow_backup_last26feb/MedGlow/ios/Pods/Flipper-Folly/folly/ssl/detail/SSLSessionImpl.cpp normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler

**Comment by nzankich** [comment]
It looks like something similar to this maybe?

https://stackoverflow.com/a/43165580/836366

Updating your xcode to latest ` 12.4` isn't a bad idea either but tbh I'm not sure if it will fix your issue
